program/convert-inches-to-feet/ConvertInchesToFeet.ts

```diff
@@ -0,0 +1,4 @@
+const ConvertInchesToFeet = (Inches: number): number => {
+    return Inches / 12;
+}
+export default ConvertInchesToFeet;
```

Comment:
can you round off the result to two decimals ?

Reply:
Yes I will do that